src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -100,6 +101,10 @@
     public static final Integer INDEX_KNN_DEFAULT_ALGO_PARAM_EF_CONSTRUCTION = 512;
     public static final Integer KNN_DEFAULT_ALGO_PARAM_INDEX_THREAD_QTY = 1;
     public static final Integer KNN_DEFAULT_CIRCUIT_BREAKER_UNSET_PERCENTAGE = 75;
+    public static final Integer KNN_DEFAULT_MODEL_CACHE_SIZE_IN_BYTES = 100000; // 100 Kb
```

Comment:
This value was more of a place holder. Im not sure what is a good value for it. I will tweak once I finish the initial train api and have concrete numbers. Its just a default so it can be updated.

Reply:
May be at least have  ```50MB``` as default? Sure I am ok to defer it till you have concrete number.